README.md

```diff
@@ -235,6 +235,7 @@ If you cannot find suitable test data on this repository, please contact us on t
       - test\_{1,2}germline.fq.gz: Synthetic raw reads file used to generate normal test data for HaplotypeCaller, simulated from chr21
       - test2\_{1,2}germline.fq.gz: Synthetic raw reads file used to generate disease test data for HaplotypeCaller
       - test*rnaseq*{1,2}.fastq.gz: reads from chr22 of sample GM12878 (SRA accession: SRX2900878) for transcriptome analysis.
+      - test*airrseq*{umi_R1,R2}.fq.gz: reads from MiSEQ sequencing of BCR data.
```

Comment:
source?

Reply:
It was a subset of in-house sequencing data, unfort. I do not have really a source for this one :(